### ISSUE TITLE
Support for R8g instance types

### ISSUE THREAD

**Issue by cPu1** [eksctl-io/eksctl#7894]
More details: https://aws.amazon.com/blogs/aws/aws-graviton4-based-amazon-ec2-r8g-instances-best-price-performance-in-amazon-ec2/

**Comment by kylehutchens** [comment]
Looking for this too, I think this [list](https://github.com/eksctl-io/eksctl/blob/7f91b9d9fb8120335b0a54b789074e11d25af617/pkg/utils/instance/instance.go#L11) just needs to get updated to include the `r8g` prefix.

**Comment by stevejr** [comment]
Any update on this?

**Comment by bryantbiggs** [comment]
this should be resolved by #8041 - cc @cheeseandcereal / @TiberiuGC

**Comment by cheeseandcereal** [comment]
Yeah, this has been shipped with the merge of #8041 which was released in 0.199.0

Closing
